### PR TITLE
Standalone mode test definitions

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
/sig node

This is for https://github.com/kubernetes/kubernetes/pull/116551/files

Discussion: https://kubernetes.slack.com/archives/C09QZ4DQB/p1678819637756759

### CODE REVIEW

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 1]
```diff
@@ -1699,6 +1699,100 @@ presubmits:
...
+        args:
+        - --root=/go/src
+        - --repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)
+        - --repo=github.com/containerd/containerd=main
+        - --upload=gs://kubernetes-jenkins/pr-logs
+        - --service-account=/etc/service-account/service-account.json
+        - --timeout=90
+        - --scenario=kubernetes_e2e
+        - --
+        - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/containerd/containerd-main/image-config.yaml
```
Comment (bobbypage) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:1730]: this image config file is using cgroupfs on cgroupv2 which not really recommended. I would suggest we use `/workspace/test-infra/jobs/e2e_node/containerd/containerd-main/systemd/image-config.yaml` and add --cgroup-driver=ssytemd to `node-test-args`, e.g. like https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-node/containerd.yaml#L1011-L1015

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 2]
```diff
@@ -1699,6 +1699,100 @@ presubmits:
...
+        - --deployment=node
+        - --gcp-zone=us-central1-b
+        - '--node-test-args=--standalone-mode=true --feature-gates=AllAlpha=true --container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/home/containerd/usr/local/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd\"]}"'
```
Comment (SergeyKanzhelev) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:1733]: key is this `--standalone-mode=true`

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml (hunk 1 above)
Comment (SergeyKanzhelev) [reply config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml]: makes sense. Fixed

**Comment by bobbypage** [comment]
thank you!

/lgtm

**Pull request merged**